### ISSUE TITLE
Unpinned tabs speed control.

### ISSUE THREAD

**Issue by Eisys** [polywock/globalSpeed#106]
I've been using this wonderful addon for a month now and love it.

However, previously I used "Video Speed Controller" and in that addon I used "remember playback speed" option which works as such;
Tab 1; select speed 1.5x
Open new Tab 2, speed is at 1.5x
Go back to Tab 1 and change speed to 1.25x.
Open new Tab 3, speed is 1.25x
Go back to Tab 2, speed is still 1.5x

So far I've had the "pin by default" option checked, because I personally don't like speed to be controlled for all tabs at once, even though that is this addon's main feature. **But,** this also is not perfect since every new tab starts at a default speed. So say for a while I'm watching normal YT videos at 2x(my default speed) and then if I start listening to music on YT, every new tab I open I have to reset the speed from 2 to 1.

So basically what I would love for this addon to have:
An extra setting under "pin by default" option, when it is unchecked*, for being able to control the speed of only the active tab and remember most recently changed speed. So it would work just like VSC.
*Unchecked because I don't really like the little "i" at the end of the speed popup in pinned mode, and when Chrome reopens from a sleep or an update, all already open tabs are unpinned anyway. Or perhaps it could apply to both unpinned and pinned modes.

Phew, I don't know if all that makes much sense.
And it's quite a niche request so it's probably not worth implementing but this is the only gripe I have with an otherwise outstanding addon.

**Comment by polywock** [comment]
Makes sense, but since all unpinned tabs share the same context, they cannot have different speed values (at least in the current implementation). If the user has "pin by default" enabled, I could make it so the new tab inherits the previous tab's context instead of the global context. Would that be okay? As for the little "i", I'll make that removable.

**Comment by Eisys** [comment]
> Makes sense, but since all unpinned tabs share the same context, they cannot have different speed values (at least in the current implementation).

Understood.

> If the user has "pin by default" enabled, I could make it so the new tab inherits the previous tab's context instead of the global context. Would that be okay?

Yes, I think that would be perfect. Best make it a toggleable setting though since I'm sure some people would prefer the current implementation.

> As for the little "i", I'll make that removable.

Yay.

**Comment by polywock** [comment]
An update, the new version (2.9.2) has been approved on the Chrome and Firefox stores. The Edge store will take a week. 

There's now a "inherit global" option once you enable "pin by default".  

![a1](https://user-images.githubusercontent.com/31208859/128643235-993d18e1-9273-47fe-b29e-7971b1476f57.png)

To disable the i. Click on the "Help" header like 10 times and it will open the command prompt. Type in "toggle i" to disable the i. 

![a3](https://user-images.githubusercontent.com/31208859/128643276-a2d7c8d8-0c2e-4234-b88f-9501dc502386.png)
